app/controllers/web/news_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Web::NewsController < Web::ApplicationController
+  def index
+    @news = Web::NewsDecorator.decorate_collection News.published.order('created_at DESC')
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.